tests/test_repository_mining.py

```diff
@@ -182,3 +184,15 @@ def test_ignore_add_whitespaces_and_changed_file():
                                    skip_whitespaces=True,
                                    single="532068e9d64b8a86e07eea93de3a57bf9e5b4ae0").traverse_commits())[0]
     assert len(commit.modifications) == 1
+
+def test_clone_repo_to():
+    tmp_folder = tempfile.TemporaryDirectory()
+    dt2 = datetime(2018, 10, 20)
+    url = "https://github.com/ishepard/pydriller.git"
+    assert len(list(RepositoryMining(
+        path_to_repo=url,
+        to=dt2,
+        clone_repo_to=tmp_folder.name).traverse_commits())) == 159
+    assert len(list(RepositoryMining(path_to_repo=os.path.join(tmp_folder.name,
+        RepositoryMining._get_repo_name_from_url(url)),
+        to=dt2).traverse_commits())) == 159
```

Comment:
What I would do to test it is to run RepositoryMining (as you do), and then check if in the `tmp_folder` there are files. Something like:
```
tmp_folder = tempfile.TemporaryDirectory()
dt2 = datetime(2018, 10, 20)
url = "https://github.com/ishepard/pydriller.git"
assert len(list(RepositoryMining(
     path_to_repo=url,
     to=dt2,
     clone_repo_to=tmp_folder.name).traverse_commits())) == 159
for dirs, subdirs, files in os.walk(tmp_folder.name):
     # check if there are files
```

Reply:
I thought of doing this initially, but changed to this approach as I thought it was more thorough in testing that there was now a fully functioning clone of the repo in the created directory. I'm happy to change it if you prefer. What are your thoughts?

Reply:
Ah now I understand your thought! Ok I see. Then leave it like this.
Though I would prefer if you change this line:
```
os.path.join(tmp_folder.name, RepositoryMining._get_repo_name_from_url(url))
```
into 
```
os.path.join(tmp_folder.name, 'pydriller')
```
it's easier to read and you don't have to call a private method.

Reply:
Okay will do